go-backend/db/user.go

```diff
@@ -6,17 +6,127 @@ import (
 	logger "github.com/sirupsen/logrus"
 )
 
+const (
+	getUserQuery = `SELECT id,
```

Comment:
do we always need the `soft_delete` columns in get response ?

Reply:
not needed each time but to check the user is currently enabled or not is not necessary?